ArduSub/radio.cpp

```diff
@@ -61,6 +61,15 @@ void Sub::init_rc_in()
     // set default dead zones
     default_dead_zones();
 
+    // initialize rc input to 1500 on control channels (rather than 0)
+    for(int i = 0; i < 7; i++) {
+    	if(i == 4) {
```

Comment:
miss a space after if. Nice trick ! I will add this to rover too !! will prevent some current problem!